src/main/java/uk/gov/hmcts/reform/sandl.snlevents/service/SessionService.java

```diff
@@ -106,6 +109,7 @@ public SessionWithHearings getSessionsWithHearingsForDates(LocalDate startDate,
         return sessionsWithHearings;
     }
 
+    //todo never used, remove?
     public List<SessionInfo> getJudgeDiaryForDates(String judgeUsername, LocalDate startDate, LocalDate endDate) {
```

Comment:
it looks like you are right, you may also check with Remi as he's the last one doing change in SessionController around this method.